charts/flagsmith/values.yaml

```diff
@@ -124,6 +126,8 @@ frontend:
     timeoutSeconds: 10
   extraInitContainers: []
   extraContainers: []
+  logging: {}
+    # format: json # Can be json or generic (plaintext).
```

Comment:
I'm not entirely sure this is true. I don't think we've added any logic for changing the logging format in the FE code?

Reply:
👍 my mistake - removed frontend references from chart & docs